frontend/components/WifiDetails.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import WifiIcon from '../components/WifiIcon';
+import WifiPopUp from '../components/WifiPopUp';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import Chip from '@material-ui/core/Chip';
+import InfoIcon from '@material-ui/icons/Info';
+
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+  row: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    gap: '1em',
+  },
+  active: {
+    color: '#FA8756',
+    fontWeight: '700',
+  },
+  wrapper: {
+    padding: '1.5em 0',
+  },
+  details: {
+    display: 'none',
+    [theme.breakpoints.up('md')]: {
+      display: 'flex',
+      gap: '1em',
+    },
+  },
+  icon: {
+    cursor: 'pointer',
+    '&:hover': {
+      opacity: 0.8,
+    },
+  },
+  chip: {
+    height: '30px',
+  },
+}));
+
+export const WifiDetails = ({ wifiNetwork }) => {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(false);
+
+  const handleDialog = () => {
+    setOpen(true);
+  };
+
+  return (
+    <>
+      <div className={`${classes.row} ${classes.wrapper}`}>
+        <div className={classes.row}>
+          <WifiIcon SIGNAL={parseInt(wifiNetwork.SIGNAL, 10)} />
+          <Typography className={wifiNetwork['IN-USE'] === '*' ? classes.active : ''}>
+            {wifiNetwork.SSID}
+          </Typography>
+        </div>
+        <div className={classes.row}>
+          <div className={classes.details}>
+            <Typography>{wifiNetwork.BSSID}</Typography>
```

Comment:
I think the `BSSID` column would look better, if they would all line up, even if the `Security` entry for the rows have different lengths.